src/order-requests/order-requests.service.ts

```
@@ -1,4 +1,234 @@
+        }
+        return {
+          productId: product.id,
+          quantity: item.quantity,
+          price: product.price, // 상품 가격 (DB에서 조회된 값)
+          notes: item.notes,
+        };
+      });
+  
+      // 3. 총 수량 계산
+      const totalAmount = orderRequestItems.reduce((sum, item) => sum + item.quantity, 0);
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)